src/fake_bpy_module/generator/translator.py

```diff
@@ -86,7 +86,13 @@ def depart_paragraph(self, _: nodes.paragraph):
         new_line_num = 0
         if len(self.status_stack) >= 1:
             status = self.status_stack[-1]
-            if status.kind in ('BULLET_LIST', 'ENUMERATED_LIST', 'NOTE', 'WARNING', 'BLOCK_QUOTE'):
+            if status.kind in (
+                "BULLET_LIST",
+                "ENUMERATED_LIST",
+                "NOTE",
+                "WARNING",
+                "BLOCK_QUOTE",
+            ):
```

Comment:
We represent enum as strings in this project.
Is it possible to change to single quote instead?
This comments should be applied other parts.

Reply:
I will use [`ruff.format.quote-style="preserve"`](https://docs.astral.sh/ruff/settings/#format_quote-style).